docs/source/dataframe-create.rst

```diff
@@ -33,6 +33,7 @@ File Formats:
     read_fwf
     from_bcolz
     from_array
+    from_dict
```

Comment:
Just a thought, not a blocker -- I think this might be better suited under a new heading "Python Dictionary" or "Python Collections", but `from_array` also doesn't fit perfectly under "file formats", so maybe we're fine.